src/utils/MeldingerPrintMarkup.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { Melding, Traad } from '../models/meldinger/meldinger';
+import { useFødselsnummer } from './customHooks';
+import {
+    eldsteMelding,
+    erJournalfort,
+    meldingstittel
+} from '../app/personside/infotabs/meldinger/utils/meldingerUtils';
+import { Element, Ingress, Normaltekst } from 'nav-frontend-typografi';
+import { datoStigende, formatterDatoMedMaanedsnavn } from './dateUtils';
+import styled from 'styled-components';
+import theme from '../styles/personOversiktTheme';
+import { formaterDato } from './stringFormatting';
+import Tekstomrade from 'nav-frontend-tekstomrade';
+
+interface Props {
+    valgtTraad: Traad;
+}
+
+const StyledTraad = styled.div`
+    display: none;
+    @media print {
```

Comment:
kanskje heller kjøre en !@media print? Tror det skrives sånn `@media not all and print`